src/index.js

```diff
@@ -1,4 +1,72 @@
 import './style.scss';
 
-// You can specify which plugins you need
-// import { Tooltip, Toast, Popover } from 'bootstrap';
\ No newline at end of file
+const leaderBoardApi = document.querySelector('#leaderboard-api');
+
+let gameID = 0;
+async function createGameId() {
+  const data = {
+    name: 'leader board',
+  };
+  let response = await fetch(`${leaderBoardApi.action}games/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+  response = await response.json();
+
+  const { result } = response;
+  const indexColumn = result.indexOf(':');
+  const indexAdded = result.indexOf('added');
+  gameID = result.substring(indexColumn + 2, indexAdded - 1);
+}
+
+async function createGame() {
```

Comment:
- [ ] As requested by the previous reviewer, kindly refactor your function to use ES6 syntax (arrow function):
### Async arrow functions look like this:
```
const foo = async () => {
  // do something
}
```